qadence/backends/braket/backend.py

```diff
@@ -45,6 +45,7 @@ def promote_parameters(parameters: dict[str, Tensor | float]) -> dict[str, float
 class Backend(BackendInterface):
     name: BackendName = BackendName.BRAKET
     supports_ad: bool = False
+    supports_adjoint: bool = False
```

Comment:
Shouldn't it be placed after the next line ?

Reply:
scaleblock derivatives are not in the paper, i computed it somehow, should be correct. there is also a test ofc

Reply:
agreed